packages/traveling-fastlane/actions/list_devices.rb

```diff
@@ -1,17 +1,44 @@
 require 'spaceship'
 require 'json'
 require 'base64'
+require 'optparse'
 require_relative 'funcs'
 
+options = {}
+OptionParser.new do |opts|
+  opts.banner = "Usage: example.rb [options]"
+
+  opts.on("-p", "--all-ios-profile-devices", "all devices that can be used for iOS profiles") do |ios_profile_devices|
+    options[:iosProfileDevices] = ios_profile_devices
+  end
+
+end.parse!
+
 $appleId, $password, $teamId = ARGV
 ENV['FASTLANE_TEAM_ID'] = $teamId
 
-def list_devices()
+def get_devices(options)
+  # if no options were specified, return all devices by default
+  if options.empty?
```

Comment:
That's a weird condition, it would be better to write it like this:
```ruby
if options.key?(:iosProfileDevices)
  # do sth
else 
  # do sth else
end
```

Reply:
i was originally thinking if someone were to add another flag, and we call the script w something like `list_devices --all-ipads --all-iphones` it'd return the union of all devices from these flags (all ipads and iphones), and if no flags are specified, it'd just return the default thing (all devices).

but maybe this is too overoptimized; ur logic looks a lot cleaner